tests/helpme/test_create.py

```
@@ -343,3 +346,308 @@ def test_determinant_spline(
+    trajectory = splinepy.BSpline(
+        degrees=[3],
+        control_points=[[0, 0, 0], [1, 0, 1], [0, 0, 2], [0, 0, 3]],
+        knot_vectors=[[0, 0, 0, 0, 1, 1, 1, 1]],
+    )
+    result = splinepy.helpme.create.swept(
+        cross_section=cross_section,
+        trajectory=trajectory,
+        anchor="parametric",
+    )
+
+    def calculate_cs_normal_vector(cs_cp, res_cp, rand):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `calculate_cs_normal_vector`

(ANN202)